packages/manager/src/features/CloudPulse/Utils/CloudPulseWidgetUtils.ts

```diff
@@ -156,28 +180,23 @@ export const generateGraphData = (props: graphDataOptionsProps) => {
     serviceType,
     status,
     unit,
-    widgetChartType,
-    widgetColor,
   } = props;
-
-  const dimensions: DataSet[] = [];
-  const legendRowsData: LegendRow[] = [];
-
-  // if the color is not found in the map, we will fallback to default color theme
-  const colors = COLOR_MAP.get(widgetColor) ?? DEFAULT;
-  let today = false;
-
+  const legendRowsData: MetricsDisplayRow[] = [];
+  // for now we will use this, but once we decide how to work with coloring, it should be dynamic
+  const dimension: { [timestamp: number]: { [label: string]: number } } = {};
+  const areas: AreaProps[] = [];
+  const colors = Object.values(Alias.Chart.Categorical);
```

Comment:
Also, here we are not considering the color configured at the widget level, lets say a widget returns 'red' color, that is not handled here, by default as of now we have only one color, is this okay

cc: @kmuddapo

Reply:
we had a discussion with Dominik yesterday related to coloring, so for now he is fine with Primary colors with 50% opacity. And for our own theme it'll be in discussion with CM team